Description: Timestamps in fpc Makefiles are hindering reproducible builds
 Don't add the Date to the header of Makefiles generated by fpcmake
Author: Paul Gevers <elbrus@debian.org>
Forwarded: no

Index: fpc/fpcsrc/utils/fpcm/fpcmwr.pp
===================================================================
--- fpc.orig/fpcsrc/utils/fpcm/fpcmwr.pp
+++ fpc/fpcsrc/utils/fpcm/fpcmwr.pp
@@ -722,7 +722,7 @@ implementation
          begin
            { Header }
            Add('#');
-           Add('# Don''t edit, this file is generated by '+TitleDate);
+           Add('# Don''t edit, this file is generated by '+Title);
            Add('#');
            if FInput.HasVariable('default_rule') then
             Add('default: '+FInput.GetVariable('default_rule',false))
